ontology/yaml/resources/subfields/subfields.yaml

```diff
@@ -160,6 +175,7 @@ descriptor:
   winter: "Method or process used during colder weather (i.e. winter season)."
   zone: "Region of building which is conditioned."
 measurement:
+  time: "measurement of time (often associated with runtime of devices)"  # Added 18/08/20
```

Comment:
a fixed time of day? if so, your units below do not work (what does a time in seconds represent?